src/components/TalentsListPage/TalentsListPage.jsx

```diff
@@ -1,6 +1,14 @@
 import React from "react";
 import { Layout } from "../Layout";
-
+import { SearchPanel } from "../TalentsListPage/SearchPanel/SearchPanel";
+import { Filter } from "./Filter";
+import { TalentsList } from "./TalentsList/TalentsList";
```

Comment:
you must add "index.js" file in the "TalentList" folder and make the appropriate export in that file. Then your import will look like:
```suggestion
import { TalentsList } from "./TalentsList";
```